pkg/commands/execution_context.go

```diff
@@ -1387,6 +1400,9 @@ func (ctx *executionContext) findTests() error {
 	if len(ctx.tests) == 0 {
 		return errors.New("there is no tests defined")
 	}
+	if len(ctx.tests) < ctx.cloudTestConfig.FailedTestsLimit {
```

Comment:
```suggestion
	if len(ctx.tests) <= ctx.cloudTestConfig.FailedTestsLimit {
```

Reply:
If we have 20 tests, we may want to set failedTestsLimit to 20, so only 19 fails allowed. It looks like valid behavior for me.

Reply:
Why treat this check result as an error, does it invalidates the testing or makes it irrelevant?
May be logrus.Warning() would be enough?